commandline/src/main/java/tech/pegasys/web3signer/commandline/valueprovider/YamlConfigFileDefaultProvider.java

```diff
@@ -162,12 +166,26 @@ private String getConfigurationValue(final OptionSpec optionSpec) {
     return String.valueOf(value);
   }
 
-  private static Stream<OptionSpec> subCommandOptions(final CommandLine subcommand) {
-    return subcommand.getCommandSpec().options().stream();
+  private static Stream<CommandLine> subCommandValues(final CommandLine c) {
+    return c.getSubcommands().values().stream();
+  }
+
+  private static Stream<OptionSpec> allSubCommandOptions(final CommandLine subcommand) {
```

Comment:
is this worth renaming too? Just not sure what ... traversalSteramOf(final CommandLine command)